public/clinics.js

```diff
@@ -85,6 +99,35 @@ function createCompanyAddressSection(index, id, name, address) {
         addressInput.value = address;
     }
     section.appendChild(addressInput);
+    section.appendChild(document.createElement('br'));
+
+    const companyPostCodeLabel = document.createElement('label');
+    companyPostCodeLabel.textContent = 'Company Postcode:';
+    section.appendChild(companyPostCodeLabel);
+    section.appendChild(document.createElement('br'));
+    const companyPostCode = document.createElement('input');
+    companyPostCode.type = 'text';
+    companyPostCode.classList.add('companyPostCode');
+    companyPostCode.name = 'companyPostCode';
+    if (name) {
+        companyPostCode.value = name;
```

Comment:
should this not be postCode instead of name. Please fix here and all other occurrences, thanks.